File: 编译原理/TinyScript-Js/src/parser/ast/Stmt.js
const ASTNode = require('./ASTNode')

/**
 * 抽象语法树抽象声明节点
 */
class Stmt extends ASTNode {
  /**
   * @param {ASTNode} _parent 
   * @param {ASTNodeTypes} _type 
   * @param {string} _label 
   */
  constructor(_parent, _type, _label) {
    super(_parent, _type, _label)
  }
}

module.exports = Stmt

File: 编译原理/TinyScript/src/main/java/parser/ast/Stmt.java
package parser.ast;

/**
 * 抽象语法树抽象声明节点
 */
public abstract class Stmt extends ASTNode {
    public Stmt(ASTNode _parent, ASTNodeTypes _type, String _label) {
        super(_parent, _type, _label);
    }
}
